cast/js/src/main/java/com/ibm/wala/cast/js/util/JSCallGraphBuilderUtil.java

```diff
@@ -194,9 +192,8 @@ public static CallGraph makeScriptCG(
       String dir, String name, CGBuilderType builderType, ClassLoader loader)
       throws IOException, IllegalArgumentException, CancelException, WalaException {
     PropagationCallGraphBuilder b = makeScriptCGBuilder(dir, name, builderType, loader);
-    CallGraph CG = b.makeCallGraph(b.getOptions());
     // dumpCG(b.getPointerAnalysis(), CG);
```

Comment:
Probably in this case the local was retained so that the next line could easily be uncommented.  Can we retain this one somehow?

Reply:
Sure, [easily done](https://github.com/wala/WALA/pull/1615/files#diff-cd424bb52b98730f78610bc25c6016af3e50283c867e1dc7678b8dac9250f1a2R195-R198). Good call.